settings/base.py

```diff
@@ -337,3 +337,5 @@ def is_environ_true(name):
 EMAIL_HOST_PASSWORD = os.environ.get('MAILGUN_SMTP_PASSWORD', '')
 EMAIL_PORT = os.environ.get('MAILGUN_SMTP_PORT', 25)
 EMAIL_USE_TLS = is_environ_true('EMAIL_USE_TLS')
+
+POSTAL_CODE_RESPONSE_CACHE_TIMEOUT = os.environ.get('CACHE_TIMEOUT', 60 * 5)
```

Comment:
@danaspiegel Per your request on card #618: 
The cache timeout should be loaded from an environment variable and should default (if not set) to 5 minutes.

Also, I would like to have it here so we can set this value to whatever needed for local settings, without the need to export new env variable each time.

What do you think?